### PR TITLE
Eng 12481 test query timeout

### PR DESCRIPTION
**Pull request by BillWhite**
Fix authentication credentials in debug.

When this is executed in debug it doesn't actually do the tests.  There is a test for isDebug() and isValgrind() which short circuits any actual testing.  But this also short circuits setting the authentication credentials.  Before, since we had started up a server already, the test couldn't create a client to shut the server down cleanly.  This caused the test to throw an exception and fail.

https://issues.voltdb.com/browse/ENG-12481

### CODE REVIEW

File: tests/frontend/org/voltdb/regressionsuites/TestQueryTimeout.java [hunk 1]
```diff
@@ -37,7 +37,12 @@
 
 public class TestQueryTimeout extends RegressionSuite {
 
-    private static final int TIMEOUT = 1000;
+    private static final double TIMEOUT_INCREASE = 50.0;
+    private static final double TIMEOUT_DECREASE = 1/500.0;
+
+    private static final int TIMEOUT_NORMAL = 1000;
```
Comment (wolffcm) [inline tests/frontend/org/voltdb/regressionsuites/TestQueryTimeout.java]: Units for this are... milliseconds I guess?  Please include the units in a comment.

File: tests/frontend/org/voltdb/regressionsuites/TestQueryTimeout.java [hunk 2]
```diff
@@ -91,7 +96,13 @@ private void truncateTables(Client client)
         truncateData(client, "R1");
     }
 
-    public void testReplicatedProcTimeout() throws IOException, ProcCallException, InterruptedException {
+    public void notestReplicatedProcTimeout() throws IOException, ProcCallException, InterruptedException {
```
Comment (wolffcm) [inline tests/frontend/org/voltdb/regressionsuites/TestQueryTimeout.java]: You'll want to remove the `no-` prefix before merging...

File: tests/frontend/org/voltdb/regressionsuites/TestQueryTimeout.java [hunk 3]
```diff
@@ -137,7 +148,13 @@ public void testReplicatedProcTimeout() throws IOException, ProcCallException, I
         }
     }
 
-    public void testPartitionedProcTimeout() throws IOException, ProcCallException, InterruptedException {
+    public void notestPartitionedProcTimeout() throws IOException, ProcCallException, InterruptedException {
```
Comment (wolffcm) [inline tests/frontend/org/voltdb/regressionsuites/TestQueryTimeout.java]: Another `no-` test-disabling prefix...

File: tests/frontend/org/voltdb/regressionsuites/TestQueryTimeout.java [hunk 4]
```diff
@@ -276,26 +297,28 @@ private void checkTimeoutIncreasedProcFailed(boolean sync, Client client, String
...
 
     private void checkTimeoutDecreaseProcFailed(boolean sync, Client client, String procName, Object...params)
             throws IOException, ProcCallException, InterruptedException {
-        checkDeploymentPropertyValue(client, "querytimeout", Integer.toString(TIMEOUT));
+        checkDeploymentPropertyValue(client, "querytimeout", Integer.toString(TIMEOUT_NORMAL));
 
         try {
             client.callProcedure(procName, params);
         }
         catch (Exception ex) {
```
Comment (wolffcm) [inline tests/frontend/org/voltdb/regressionsuites/TestQueryTimeout.java]: You could change this to expect a `ProcCallException` like you did above?

File: tests/frontend/org/voltdb/regressionsuites/TestQueryTimeout.java [hunk 5]
```diff
@@ -398,6 +425,12 @@ private void checkIndividualProcTimeout(Client client, boolean isAdmin)
     }
 
     public void testIndividualProcTimeout() throws IOException, ProcCallException, InterruptedException {
+        // If the authentication information is
+        // incorrect, shutdown will not succeed.
+        // So, we need to set this in any case.
+        // Feel free to reset it later on.
+        m_username="adminUser";
```
Comment (wolffcm) [inline tests/frontend/org/voltdb/regressionsuites/TestQueryTimeout.java]: In this case, these member variables seem to be set to the same values immediately below... delete lines 440-441?

File: tests/frontend/org/voltdb/regressionsuites/TestQueryTimeout.java (hunk 2 above)
Comment (BillWhite) [inline tests/frontend/org/voltdb/regressionsuites/TestQueryTimeout.java]: Done.  (If I had a nickel.)

File: tests/frontend/org/voltdb/regressionsuites/TestQueryTimeout.java (hunk 1 above)
Comment (BillWhite) [inline tests/frontend/org/voltdb/regressionsuites/TestQueryTimeout.java]: Done.

File: tests/frontend/org/voltdb/regressionsuites/TestQueryTimeout.java (hunk 3 above)
Comment (BillWhite) [inline tests/frontend/org/voltdb/regressionsuites/TestQueryTimeout.java]: Done.

File: tests/frontend/org/voltdb/regressionsuites/TestQueryTimeout.java (hunk 5 above)
Comment (BillWhite) [inline tests/frontend/org/voltdb/regressionsuites/TestQueryTimeout.java]: Done.

**Pull request merged**